ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -49,6 +50,10 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+            this.isCreating = flatRouteData.data.hasOwnProperty('creating') && flatRouteData.data['creating'];
+
+            if(this.isCreating)
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
@jsight IDK. How would the deleting affect the user at all? It happens in the background, async, and the user doesn't really need it to happen. Nothing blocks him from creating the project.